dataplaneapi.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: dataplaneapi
+  version: 2.7.2
+  epoch: 0
+  description: HAProxy Data Plane API
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - build-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/haproxytech/dataplaneapi
+      branch: v${{package.version}}
```

Comment:
It looks like there is no branch for 2.7.2, that's the issue.

Reply:
Youre right :))